tasks/main.yml

```diff
@@ -7,11 +7,14 @@
   register: sestatus
   failed_when: false
   ignore_errors: true
```

Comment:
Remove `ignore_errors: true`. It's redundant, since we have `failed_when: false` above it.